src/aspire/config.ini

```diff
@@ -1,5 +1,6 @@
 [common]
-cupy = 0
+# numeric module to use - one of numpy/cupy
+numeric = numpy
```

Comment:
@garrettwrong I remember we had some discussions about how we wanted to structure the options, but I don't recall the details. Is this in line?